presentations/basic-types/slides.adoc

```diff
@@ -6,17 +6,17 @@ link:./index.html[Table of Contents]
 
 Rust comes with all standard int types, with and without sign
 
-- i8, u8
-- i16, u16
-- i32, u32
-- i64, u64
-- i128, u128
+- `i8`, `u8`
+- `i16`, `u16`
+- `i32`, `u32`
+- `i64`, `u64`
+- `i128`, `u128`
```

Comment:
I wonder if it's useful to provide an example in a separate slide on how one can specify different numeric literals (similar to the chars example code). It is touched on in one example that uses a type suffix, e.g. `3_i64`, I didn't find any further examples so far (I might have missed it).

Something like:

```rust
123_456  // underscore as separator
0x12 // prefix 0x to indicate hex value
0o23 // prefix 0o to indicate octal value
0b0001 // prefix 0b to indicate binary value
b'a' // A single byte character
```

What do you think?

If you think that's useful I can prepare a separate PR to add an extra slide. :)

Reply:
This is fantastic idea. I will add this slide myself and you can review it if you want.

Reply:
Done.